modules/gapi/test/internal/gapi_int_recompilation_test.cpp

```diff
@@ -151,9 +152,9 @@ TEST(GComputationCompile, FluidReshapeSwitchToUpscaleFromDownscale)
     cv::resize(in_mat2, cv_out_mat2, szOut);
     cv::resize(in_mat3, cv_out_mat3, szOut);
 
-    EXPECT_EQ(0, cvtest::norm(out_mat1, cv_out_mat1, NORM_INF));
-    EXPECT_EQ(0, cvtest::norm(out_mat2, cv_out_mat2, NORM_INF));
-    EXPECT_EQ(0, cvtest::norm(out_mat3, cv_out_mat3, NORM_INF));
+    EXPECT_EQ(1, cvtest::norm(out_mat1, cv_out_mat1, NORM_INF));
+    EXPECT_EQ(1, cvtest::norm(out_mat2, cv_out_mat2, NORM_INF));
+    EXPECT_EQ(1, cvtest::norm(out_mat3, cv_out_mat3, NORM_INF));
```

Comment:
Please add a comment here that `1` is here because of Fluid resize used instead OpenCV's